### DISCUSSION TITLE
Run Entire Collection?

### DISCUSSION THREAD

**Discussion by alexbot89** [hoppscotch/hoppscotch#2266] [Q&A] [answered]
I am currently evaluating hoppscotch but would like to know if several features are currently in dev or are currently not in the app.

My use case is to:
1. Run an entire collection containing requests, each with tests or pre-request scripts attached
2. Run the entire collection through CLI, activating the tests attached to the request.

I looked at the demo site, but could not find an option to run the entire collection.

Postman has a cli application called newman,
https://www.npmjs.com/package/newman

Is there a similar cli app for hoppscotch, it has to be able to evaluate the tests as well?
The closest thing I see is.
https://github.com/hoppscotch/hopp-cli
However this only outputs the status code and not the body of the response, I also don't know if it activates the test in the request.

**Comment by AndrewBastin** [comment, marked as answer]
Hi @alexbot89,

We are soft testing a revamped Hoppscotch CLI (https://www.npmjs.com/package/@hoppscotch/cli). You can read about it in our docs as well (https://docs.hoppscotch.io/cli). All the other CLI projects are going to be marked deprecated in the near future.

Please give your feedback!

**Reply by alexbot89** [reply]
Wow this looks quite promising, I will give this a try, thank you.

**Comment by arnaduga** [comment]
https://docs.hoppscotch.io/cli is a dead link....

Website problemm or abandonned feature? Any solution to run an entire collection?

**Reply by arnaduga** [reply]
Found this https://docs.hoppscotch.io/documentation/clients/cli/, forget about my message ^^
